Sources/App/Core/Github.swift

```diff
@@ -15,8 +15,14 @@ enum Github {
         var description: String?
         var forksCount: Int
         var license: License?
-        var stargazersCount: Int
+        var name: String?
+        var owner: Owner?
         var parent: Parent?
+        var stargazersCount: Int
+
+        struct Owner: Decodable, Equatable {
+            var login: String?
+        }
```

Comment:
It'd probably be safe to parse the owner/name out of the existing `fullName` field but since it's easy to just get the original fields I've added those instead.

Reply:
Gotcha, looking at the GitHub API docs login seems a strange name for it but it matches the API response. I agree since we've already got it coming down, it's better to read it from the API response than parse it.